lib/zookeeper.js

```diff
@@ -45,7 +45,7 @@ const create = (con, p, cb) => {
  * @param {mkdirCb} callback
  */
 const mkdirp = (con, p, callback) => {
-    const dirs = normalize(p).split('/').slice(1); // remove empty string at the start.
+    const dirs = normalize(p).split(sep).slice(1); // remove empty string at the start.
```

Comment:
On line 53, there is also a `normalize` call. I think that will cause trouble in Windows too.

A possible solution is to use the `posix` feature (see animations).

Windows:
![windows-posix](https://user-images.githubusercontent.com/301286/153818734-6ae6fce8-333d-4fed-993f-7a9bb3f95785.gif)


Mac OS X:
![mac-posix](https://user-images.githubusercontent.com/301286/153818768-58b38b45-d743-4230-ad3f-cb6a8e729010.gif)

Reply:
I also tested it, posix.normalize is great